javascript/ql/lib/semmle/javascript/frameworks/Fetch.qll

```diff
@@ -0,0 +1,19 @@
+/**
+ * Provides an additional flow step that propagates data from the receiver of fetch response methods.
+ */
+
+private import javascript
+
+/**
+ * An additional flow step that propagates data from the receiver of fetch response methods
+ * (like "json", "text", "blob", and "arrayBuffer") to the call result.
+ */
+class Fetch extends DataFlow::AdditionalFlowStep {
```

Comment:
The class here is a duplicate of the one in `ClientRequests`.

Actually let's go ahead and remove this whole file as `ClientRequests.qll` is a good enough place for the model of `fetch`.